src/Driver/AbstractMySQLDriver.php

```diff
@@ -69,7 +70,11 @@ public function createDatabasePlatformForVersion($version)
                     );
                 }
 
-                return new MySQL80Platform();
+                if (version_compare($version, '8.0.13', '<')) {
+                    return new MySQL80Platform();
+                }
```

Comment:
We have tests that make sure we pick the right platform for a given version string. Please update that test for your new platform class.

Reply:
Ok. Done